tests/test_integration.py

```diff
@@ -0,0 +1,59 @@
+from time import sleep
+from unittest.mock import call
+
+import pytest
+from mopidy.models import Track
+
+import mopidy_mpris
+
+from . import DictFields
+
+
+def player_call(can_play, can_go_next, can_go_previous):
+    vals = DictFields(
+        {
+            "CanPlay": can_play,
+            "CanGoNext": can_go_next,
+            "CanGoPrevious": can_go_previous,
+        }
+    )
+    return call("org.mpris.MediaPlayer2.Player", vals, [])
+
+
+def assert_called(can_play, can_go_next, can_go_previous):
+    # Beware the multithreading! Wait for the signal to get emitted.
+    sleep(0.1)
+    mopidy_mpris.interface.Interface.PropertiesChanged.assert_has_calls(
+        [player_call(can_play, can_go_next, can_go_previous)]
+    )
+    mopidy_mpris.interface.Interface.PropertiesChanged.reset_mock()
```

Comment:
I think this would read better with `player_call` inlined and by requiring the arguments to `assert_called` to be kwargs:

```suggestion
def assert_called(*, can_play, can_go_next, can_go_previous):
    # Beware the multithreading! Wait for the signal to get emitted.
    sleep(0.1)
    mopidy_mpris.interface.Interface.PropertiesChanged.assert_has_calls(
        [
            call(
                "org.mpris.MediaPlayer2.Player",
                DictFields({
                    "CanPlay": can_play,
                    "CanGoNext": can_go_next,
                    "CanGoPrevious": can_go_previous,
                }),
                [],
            ),
        ]
    )
    mopidy_mpris.interface.Interface.PropertiesChanged.reset_mock()
```